devenv/docker/blocks/auth/jwt_proxy/docker-compose.yaml

```diff
@@ -6,7 +6,7 @@
       POSTGRES_USER: keycloak
       POSTGRES_PASSWORD: password
     volumes:
-      - ./docker/blocks/jwt_proxy/cloak.sql:/docker-entrypoint-initdb.d/cloak.sql
+      - ./docker/blocks/auth/jwt_proxy/cloak.sql:/docker-entrypoint-initdb.d/cloak.sql
```

Comment:
@linoman had to fix this here. There's always one that slips through the cracks :smile:

Reply:
I believe I forgot about the volumes. I just created a PR to address this.

https://github.com/grafana/grafana/pull/54828